recipes/_package.rb

```diff
@@ -16,18 +17,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+class ::Chef::Recipe
```

Comment:
You want to include this via the library not not via the recipe. It makes it a lot more clear that you're injecting into a global space.

From chef-client cookbook
Chef::DSL::Recipe.send(:include, Opscode::ChefClient::Helpers)
Chef::Resource.send(:include, Opscode::ChefClient::Helpers)